### PR TITLE
feat(SRVKP-3655): Switch reoccurring runs to upstream and add upstream on-demand jobs

### PR DESCRIPTION
**Pull request by jhutar**

### CODE REVIEW

**Review by vdemeester — APPROVED** [review]
/lgtm

**Comment by jhutar** [comment]
/pj-rehearse pull-ci-openshift-pipelines-performance-master-scaling-pipelines-upstream-nightly pull-ci-openshift-pipelines-performance-master-scaling-pipelines-upstream-stable

**Review by khrm — COMMENTED** [review]
/lgtm 
/approve

**Comment by jhutar** [comment]
/pj-rehearse pull-ci-openshift-pipelines-performance-master-scaling-pipelines-upstream-nightly pull-ci-openshift-pipelines-performance-master-scaling-pipelines-upstream-stable

**Comment by jhutar** [comment]
/pj-rehearse pull-ci-openshift-pipelines-performance-master-scaling-pipelines-upstream-stable pull-ci-openshift-pipelines-performance-master-scaling-pipelines-upstream-nightly

**Comment by jhutar** [comment]
/pj-rehearse pull-ci-openshift-pipelines-performance-master-scaling-pipelines-upstream-stable

**Comment by jhutar** [comment]
/pj-rehearse pull-ci-openshift-pipelines-performance-master-scaling-pipelines-upstream-nightly

**Comment by jhutar** [comment]
/pj-rehearse pull-ci-openshift-pipelines-performance-master-scaling-pipelines-upstream-stable

**Comment by jhutar** [comment]
/pj-rehearse pull-ci-openshift-pipelines-performance-master-scaling-pipelines-upstream-nightly

**Comment by jhutar** [comment]
/pj-rehearse pull-ci-openshift-pipelines-performance-master-scaling-pipelines-upstream-stable pull-ci-openshift-pipelines-performance-master-scaling-pipelines-upstream-nightly

File: ci-operator/step-registry/openshift-pipelines/scaling-pipelines/openshift-pipelines-scaling-pipelines-commands.sh [hunk 1]
```diff
@@ -48,13 +48,15 @@ else
 fi
 
 cd "$(mktemp -d)"
-git clone --branch master https://github.com/openshift-pipelines/performance.git .
+git clone --branch jhutar https://github.com/openshift-pipelines/performance.git .
```
[the remark this answers is not in the document]
Comment (jhutar) [reply ci-operator/step-registry/openshift-pipelines/scaling-pipelines/openshift-pipelines-scaling-pipelines-commands.sh:51]: Damn, sorry, forgot to remove it

[the remark this answers is not in the document]
Comment (jhutar) [reply ci-operator/step-registry/openshift-pipelines/scaling-pipelines/openshift-pipelines-scaling-pipelines-commands.sh]: Please check now.

**Review by vdemeester — COMMENTED** [review]
/lgtm

**Pull request merged**